sdk/maps/maps-route-rest/recordings/browsers/get_route_range/recording_should_accept_latlon_and_return_reachable_range.json

```diff
@@ -0,0 +1,252 @@
+{
+  "Entries": [
+    {
+      "RequestUri": "https://atlas.microsoft.com/route/range/json?query=50.97452%2C5.86605\u0026timeBudgetInSec=600\u0026api-version=1.0",
+      "RequestMethod": "GET",
+      "RequestHeaders": {
+        "Accept": "application/json",
+        "Accept-Encoding": "gzip, deflate, br",
+        "Accept-Language": "en-US",
+        "Connection": "keep-alive",
+        "Referer": "http://localhost:9876/",
+        "sec-ch-ua": "",
+        "sec-ch-ua-mobile": "?0",
+        "sec-ch-ua-platform": "",
+        "Sec-Fetch-Dest": "empty",
+        "Sec-Fetch-Mode": "cors",
+        "Sec-Fetch-Site": "same-site",
+        "subscription-key": "FlEIfedJiCE9pnEXvoU6-HWVxB1SP-ddYhEBfV5K3xo",
```

Comment:
Subscription key shouldn't show up in json files. This is a leak of credential. Please redact subscription key and rotate your key.

Reply:
Oh, that's a huge mistake. Thanks for the reminder. I've rotated the key. The fixed is in 96781dc3df277391148097eb60c21688978091e5